ci-operator/step-registry/hypershift/kubevirt/conformance/hypershift-kubevirt-conformance-workflow.yaml

```diff
@@ -21,14 +21,18 @@ workflow:
     env:
       COMPUTE_NODE_TYPE: c5n.metal
       CONTROL_PLANE_INSTANCE_TYPE: m4.xlarge
+      TEST_CSI_DRIVER_MANIFEST: kubevirt-driver.yaml
     pre:
     - chain: ipi-aws-ovn-pre
     - chain: ipi-install
+    - ref: storage-conf-csi-kubevirt
+    - ref: hypershift-kubevirt-install-odf
     - ref: hypershift-kubevirt-install
     - ref: hypershift-install
     - chain: hypershift-kubevirt-create
     test:
     - chain: hypershift-conformance
+    - chain: hypershift-kubevirt-run-csi-e2e
```

Comment:
It might be better if we had one step registry `hypershift-conformance` that validated everything we need tested. Otherwise, kubevirt isn't aligned with how conformance for the other platforms work.

for example, other configs managed by other teams expect the `hypershift-conformance` chain to contain everything related to validating hypershift/kubevirt. there are two other references to `hypershift-conformance` other than this one which will not validate csi as it is written now. 


```
$ grep -r -n hypershift-conformance ci-operator/step-registry/* | grep kubevirt
ci-operator/step-registry/hypershift/kubevirt/conformance/hypershift-kubevirt-conformance-workflow.yaml:34:    - chain: hypershift-conformance
ci-operator/step-registry/hypershift/kubevirt/baremetalds/conformance/hypershift-kubevirt-baremetalds-conformance-workflow.yaml:27:    - chain: hypershift-conformance
ci-operator/step-registry/hypershift/mce/kubevirt/conformance/hypershift-mce-kubevirt-conformance-workflow.yaml:15:    - chain: hypershift-conformance
```

Reply:
The thing is, the kubevirt-csi stuff is special in that it is not managed by the storage-operator (yet) and thus has special flow that doesn't apply to the other providers. If I merge this stuff into the hypershift-conformance step, then all the other providers will get it as well, and I don't think that is what we want.